### ISSUE TITLE
Add Latest Things opt-out to Account Settings

### ISSUE THREAD

**Issue by kaberett** [dreamwidth/dw-free#2280]
Possibly under privacy?

The console command is `set latest_optout yes` and there are periodic support requests about this (most recently https://www.dreamwidth.org/support/see_request?id=38071).

https://www.dreamwidth.org/support/faqbrowse?faqid=104 will need updating (and possibly a new FAQ creating?)

**Comment by kaberett** [comment]
... or possibly, if this is too Options Proliferation, add instructions on how to opt out to the Latest Things page itself?

**Comment by zorkian** [comment]
The reason it's not documented or linked is a policy decision I
personally made when I implemented the feature. It wasn't forgotten
or anything.
I think it might be reasonable to tie this to the 'don't index my
journal' setting, which puts up Robots.txt fields so well behaved bots
won't index. If someone has that enabled, I'd be fine if we wanted to
blacklist their content from Latest Things.
Thoughts?

**Comment by rahaeli** [comment]
I like that; minimizes settings to juggle. Can you tell how many people have LT optout set but not the robots exclusion?

**Comment by zorkian** [comment]
Yes, let me see...

Prop 60 is block robots, 50 is latest_optout.

```
mysql> select count(*) from userproplite2 where upropid = 50 and userid in (select userid from userproplite2 where upropid = 60);
+----------+
| count(*) |
+----------+
|      610 |
+----------+
1 row in set (6.43 sec)
```

Looks like about 600 people have LT optout set but allow robots. For comparison, there are 24,473 people with the optout set and 38,663 people with robots blocking set.

**Comment by zorkian** [comment]
Whoops, I think I reversed my logic. I think that's people who have both set? Hold on.

**Comment by zorkian** [comment]
Yeah I inversed it. It turns out they're pretty disjoint sets -- most users have one or the other. I don't know if this is because they don't know about both or what, though.

I still think I'm inclined to combine the options and make them do both but I don't feel super strongly about it or anything.

**Comment by rahaeli** [comment]
Yeah let's do that, and make it so if people had either one set, they have them both set after. but maybe hold off on doing it until the next push when I'm more likely to be able to do the heavy typing needed for a news post so we can let people know more or less immediately to change the setting of that want?

**Comment by azurelunatic** [comment]
I recall some explicit discussion Back In The Day from some of the people who know about the Latest Things opt-out and who have blocked robots: at the time, the small userbase and friend-of-a-friend-of-a... nature of the site meant they were comfortable with other Dreamwidth users discovering them easily, but were generally against the whole internet finding it easy to show up. 

This opinion may have shifted during the intervening _oh holy shit_ 9-ish years of flight.
